Directory.Build.targets

```diff
@@ -10,10 +10,11 @@
     Comment out this section if you need to update Resource.designer.cs files.
     See DEVELOPMENT.md#Android for details.
   -->
-  <PropertyGroup Condition="'$(TargetPlatformIdentifier)' == 'Android' and '$(AndroidApplication)' != 'true'">
+  <PropertyGroup Condition="($(TargetFramework.StartsWith('MonoAndroid')) or '$(TargetPlatformIdentifier)' == 'Android') and '$(AndroidApplication)' != 'true'">
```

Comment:
Is this supported in MonoAndroid?

Reply:
Yes, once I added this it started compiling for me locally